components/library/objfw/objfw.license

```diff
@@ -0,0 +1,3 @@
+ObjFW is triple-licensed. It can be used either under the terms described in
+the file LICENSE.QPL, LICENSE.GPLv3 or LICENSE.GPLv2, as distributed with
```

Comment:
We have the ability to include all three licenses in the package, we should do so. There can be many license actions not just one. The variable in the makefile just populates the p5m file during build.

Reply:
So I should set the variable to a space-separated list of files then?